docs/types/mapping-types.rst

```diff
@@ -52,6 +58,34 @@ each ``_KeyType``, recursively. For example with a mapping:
         }
     }
 
+The example below uses a mapping type inside another mapping type. The important
+point to remember are that the mapping inside the mapping is not the return value,
+but the ``bool`` inside it is.
+
+::
+
+    pragma solidity >=0.4.0 <0.7.0;
+
+    contract MappingExample {
```

Comment:
Do you think a full credit / allowance example would be too long?

Reply:
@chriseth Possibly, but how much do you think the full example relates to mappings and how they work? Or maybe a reduced example of what you suggest that highlights those aspects? I assume based on the Zeppelin link from above?

Reply:
I think you can use the following functions and also slim them down a little:
 - approve
 - transferFrom 
 - allowances